pkg/parser/parser.y

```diff
@@ -6525,6 +6529,7 @@ UnReservedKeyword:
 |	"AFTER"
 |	"ALWAYS"
 |	"AVG"
+|   "BDR"
```

Comment:
more indent？

Reply:
fixed in https://github.com/pingcap/tidb/pull/48504/commits/bd24d7266455c71f8e43958760ab15a27432eb82